tracer/test/Datadog.Trace.TestHelpers/CSharpTracingIntegrationRules.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="CSharpTracingIntegrationRules.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Sampling;
+using Datadog.Trace.Vendors.StatsdClient;
+
+namespace Datadog.Trace.TestHelpers
+{
+#pragma warning disable SA1601 // Partial elements should be documented
+    public static partial class CSharpTracingIntegrationRules
```

Comment:
The C# rules are incomplete right now and the names are not final, but that can be updated once I get feedback about whether to go with the F# logic or the C# logic. Having both would be redundant

Reply:
I removed the F# code in favor of C# which should be more maintainable in this repo